src/tmpfiles/tmpfiles.c

```diff
@@ -3973,16 +3973,19 @@ static int help(void) {
         if (r < 0)
                 return log_oom();
 
-        printf("%s [OPTIONS...] [CONFIGURATION FILE...]\n"
-               "\n%sCreates, deletes and cleans up volatile and temporary files and directories.%s\n\n"
+        printf("%1$s COMMAND [OPTIONS...] [CONFIGURATION FILE...]\n"
+               "\n%2$sCreates, deletes, and cleans up files and directories.%3$s\n\n"
+               "Commands\n"
+               "     %2$s--create%3$s               Create files and directories\n"
+               "     %2$s--clean%3$s                Clean up files and directories\n"
+               "     %2$s--remove%3$s               Remove files and directories\n"
```

Comment:
i'd drpo the coloring for this, I think the sections are good enough for separating this out.